src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Perhaps a header comment can be added here to specify the purpose of this interface.
Recall from tutorial that an interface is a behavioural contract. 
The header comment could perhaps be `Represents a printable entity .......`.